codes/quantum/qubits/bravyibaconshor.yml

```diff
@@ -0,0 +1,24 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: bravyi_bacon_shor
+physical: qubits
+logical: qubits
+
+name: 'Bravyi-Bacon-Shor (BBS) code'
+introduced: '\cite{arXiv:1207.1443v2}'
+
+description: ''
+
+protection: 'Detects errors on \(d-1\) qubits, corrects errors on \(\left\lfloor (d-1)/2 \right\rfloor\) qubits.'
```

Comment:
Are you sure this is true? If not, let's remove it. I'm fine wiht leaving this as a stub since I've asked you to do a lot of work on the Bacon-Shor part.